lib/jest-helpers.js

```diff
@@ -15,38 +15,57 @@ const sassRender = util.promisify(sass.render)
 const configPaths = require('../config/paths.json')
 const { componentNameToMacroName } = require('./helper-functions.js')
 
-nunjucks.configure([configPaths.src, configPaths.components], {
+const nunjucksEnv = nunjucks.configure([configPaths.src, configPaths.components], {
   trimBlocks: true,
   lstripBlocks: true
 })
 
 /**
- * Render a component's macro for testing
- * @param {string} componentName
- * @param {string} params parameters that are used in the component macro
- * @param {any} children any child components or text, pass the children to the macro
- * @returns {function} returns cheerio (jQuery) instance of the macro for easy DOM querying
+ * Render the raw HTML for a component
+ *
+ * @param {String} componentName
+ * @param {Object} options options to pass to the component macro
+ * @param {String} callBlock if provided, the macro is called using the Nunjucks
+ *   call tag, with the callBlock passed as the contents of the block
+ * @returns {String} HTML rendered by the macro
  */
-function render (componentName, params, children = false) {
-  if (typeof params === 'undefined') {
+function renderHtml (componentName, options, callBlock = false) {
+  if (typeof options === 'undefined') {
     throw new Error('Parameters passed to `render` should be an object but are undefined')
   }
 
   const macroName = componentNameToMacroName(componentName)
-  const macroParams = JSON.stringify(params, null, 2)
+  const macroParams = JSON.stringify(options, null, 2)
 
   let macroString = `{%- from "${componentName}/macro.njk" import ${macroName} -%}`
 
   // If we're nesting child components or text, pass the children to the macro
   // using the 'caller' Nunjucks feature
-  if (children) {
-    macroString += `{%- call ${macroName}(${macroParams}) -%}${children}{%- endcall -%}`
+  if (callBlock) {
+    macroString += `{%- call ${macroName}(${macroParams}) -%}${callBlock}{%- endcall -%}`
   } else {
     macroString += `{{- ${macroName}(${macroParams}) -}}`
   }
 
-  const output = nunjucks.renderString(macroString)
-  return cheerio.load(output)
+  return nunjucksEnv.renderString(macroString)
+}
+
+/**
+ * Render component into Cheerio API
+ *
+ * Allows us to interrogate the output of the macro using a jQuery-like syntax
+ *
+* @param {String} componentName
```

Comment:
Just a little misalligned comment 😄 

```suggestion
 * @param {String} componentName
```